libdnf5/base/base.cpp

```diff
@@ -38,45 +39,6 @@ along with libdnf.  If not, see <https://www.gnu.org/licenses/>.
 
 namespace fs = std::filesystem;
 
-namespace {
```

Comment:
I am not 100% sure but the commit might be merged with previous commit or is there any reason why you would prefer the split?